<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(111, 111, 111);</string>
  </property>
  <widget class="QLabel" name="TITLE">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>14</pointsize>
     <weight>10</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 87 14pt &quot;Segoe UI Black&quot;;
</string>
   </property>
   <property name="text">
    <string>WELCOME TO YT VIDEO FETCHER</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="URL">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>PASTE THE URL :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="URL_GET">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(85, 255, 255);
font: 87 8pt &quot;Segoe UI Black&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="path">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>CHOOSE PATH TO SAVE:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="path_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
background-color: rgb(185, 185, 185);
</string>
   </property>
   <property name="text">
    <string>CHOOSE DIRECTORY /..</string>
   </property>
  </widget>
  <widget class="QLabel" name="path_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>VIDEO QUALITY : </string>
   </property>
  </widget>
  <widget class="QComboBox" name="choice_combo">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(185, 185, 185);
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(255, 255, 255);
font: 87 8pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>MAX RESOLUTION</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LOW RESOLUTION</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AUDIO ONLY</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="dwnld">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
color: rgb(255, 255, 255);
background-color: rgb(234, 11, 25);
</string>
   </property>
   <property name="text">
    <string>DOWNLOAD</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>480</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string>MADE BY SIRJAN CHAWLA</string>
   </property>
  </widget>
  <widget class="QLabel" name="url_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="path_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="prog_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="resetbtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>430</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
color: rgb(255, 255, 255);
background-color: rgb(234, 11, 25);
</string>
   </property>
   <property name="text">
    <string>RESET</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
